.clangformatignore

```diff
@@ -1,7 +1,7 @@
 apps/uvc-app/include
 apps/uvc-app/lib
 bindings/python/pybind11
-dependencies/
+dependencies/third_party
```

Comment:
the actual name of the folder is "third-party" with  a dash instead of underscore

Reply:
that's why CI is failing

Reply:
Solved